_includes/scenarios/integrate/identityattribute-introduction.md

```diff
@@ -0,0 +1,23 @@
+An [Identity Attribute]({% link _docs_integrate/attribute-values.md %}#identity-attributes) is a specific piece of information associated with an identity (person or entity) in the context of the networked ecosystem. Attribute values can take on different types that define the format, validation rules, and presentation information for the stored data.
+
+Identity attributes are critical for managing and sharing information within the networked ecosystem and can include both simple, atomic attribute value types (e.g., email address, display name, phone number) and more complex types (e.g., street address, personal name). These attributes may also contain other attribute values, depending on their specific requirements.
```

Comment:
- Cardinality of IdentityAttributes (e.g. multiple given names or addresses)
- The semantics behind attributes count.